hawkmoth/__init__.py

```diff
@@ -39,21 +39,18 @@ class CAutoBaseDirective(SphinxDirective):
     _docstring_types = None
 
     def __display_parser_diagnostics(self, errors):
-        # Map parser diagnostic level to Sphinx verbosity and level name
+        # Map parser diagnostic level to Sphinx level name
         log_level_map = {
-            logging.DEBUG: (3, 'DEBUG'),
-            logging.INFO: (3, 'INFO'),
-            logging.WARNING: (1, 'WARNING'),
-            logging.ERROR: (0, 'ERROR'),
-            logging.CRITICAL: (0, 'CRITICAL'),
+            logging.DEBUG: 'DEBUG',
+            logging.INFO: 'VERBOSE',
```

Comment:
I dropped this change in my suggestion by mistake only, but you get the idea. Feel free to add it back, it makes sense to me.